src/main/webapp/app/overview/course-archive/course-archive.component.ts

```
@@ -0,0 +1,151 @@
+        return courses.sort((courseA, courseB) => (courseA.title ?? '').localeCompare(courseB.title ?? ''));
+    }
+
+    getUniqueSemesterNamesSorted(courses: CourseForArchiveDTO[]): string[] {
+        return (
+            courses
+                .map((course) => course.semester ?? '')
+                // filter down to unique values
+                .filter((course, index, courses) => courses.indexOf(course) === index)
+                .sort((semesterA, semesterB) => {
+                    // Parse years in base 10 by extracting the two digits after the WS or SS prefix
+                    const yearsCompared = parseInt(semesterB.slice(2, 4), 10) - parseInt(semesterA.slice(2, 4), 10);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)